### DISCUSSION TITLE
CSV Product Import Multilanguage

### DISCUSSION THREAD

**Discussion by lx-maxhaslehner** [swellstores/community#508] [Help] [answered]
Hi Community, 

I am fairly new to Swell and am trying to import my products into the dashboard via CSV Import. I am using a multi locale setup and was wondering if there is an option to directly import the translations for properties e.g the name for multiple locales with this file.
I was searching in the documentation but unfortunately could not find information according to that.. 

Looking forward to hear from you, 
Max 🙂

**Comment by logeshswell** [comment, marked as answer]
Hello Max, we have a few limitations with the CSV import and it is one of them. So, you shall choose backend APIs to experience endless flexibility.
